doc/source/installation/apt_repo.rst

```diff
@@ -146,6 +145,44 @@ This will install all the packages from the bundle. Another option is to downloa
 
   When installing packages manually like this, you'll need to make sure to resolve all the dependencies and install missing packages yourself. Following packages will need to be installed before you can manually install Percona Server: ``mysql-common``, ``libjemalloc1``, ``libaio1`` and ``libmecab2``
 
+.. rubric:: AppArmor settings
+
+Debian systems may enable AppArmor service on default installations. This security service controls how applications access the file system by creating application profiles.
+
+If you must have AppArmor enabled, you can extend the MySQL profile. You can edit the profile, which is stored in the following locations:
+
+* /etc/apparmor.d/usr.sbin.mysqld
+* /etc/apparmor.d/local/usr.sbin.mysqld
+
+You can edit the profile for this server in the ``local`` file. The profile gives read (r) access to a directory and read, write, and lock access (rwk), controlled by the contents recursively, controlled by **.
```

Comment:
"The profile gives read (r) access to a directory and read, write, and lock access (rwk), controlled by the contents recursively, controlled by **."
This sentence is unclear: to what is rwk access granted?

Reply:
Fixed.